accounts/scwallet/hub.go

```diff
@@ -153,7 +153,7 @@ func (hub *Hub) setPairing(wallet *Wallet, pairing *smartcardPairing) error {
 
 // NewHub creates a new hardware wallet manager for smartcards.
 func NewHub(daemonPath string, scheme string, datadir string) (*Hub, error) {
-	context, err := pcsc.EstablishContext(pcsc.ScopeSystem)
+	context, err := pcsc.EstablishContext(daemonPath, pcsc.ScopeSystem)
```

Comment:
OMG, we lagged with `go-libpcsclight` version for a long time... This was probably a problem just after migration to go modules. (I remember, that in upstream PR, they notified me about this version problem (and it caused this change in API))

Reply:
Hm. I've seen this conflict before, so that would make sense.